src/components/OrdersWidget/index.tsx

```diff
@@ -96,16 +102,26 @@ const OrdersWidget: React.FC = () => {
       }
     })
 
+    allPendingOrders.forEach(order => {
```

Comment:
Feels like you could extract a function to split the orders.
Something like: 

```
function splitOrders(sourceOrders:AuctionElement[], ordersState: state, stateKey: string):void {
  ...
}
```

Reply:
https://github.com/gnosis/dex-react/issues/657